src/menus/icon-menu.jsx

```diff
@@ -114,12 +108,6 @@ const IconMenu = React.createClass({
      * horizontal: [left, center, right].
      */
     targetOrigin: PropTypes.origin,
-
-    /**
-     * Sets the delay in milliseconds before closing the
-     * menu when an item is clicked.
-     */
-    touchTapCloseDelay: React.PropTypes.number,
```

Comment:
Right, maybe. Whatever it was for, it doesn't seem like there is any alternative to replace its behavior, is there? Are we just not going to support this behavior anymore?